core/src/main/scala/kafka/server/ControllerApis.scala

```diff
@@ -151,6 +150,11 @@ class ControllerApis(val requestChannel: RequestChannel,
     handleRaftRequest(request, response => new FetchSnapshotResponse(response.asInstanceOf[FetchSnapshotResponseData]))
   }
 
+  // This API is not wired in yet as the controller does not implement the
+  // metadata completely.
+  // Leaving the code in to be completed and wired in the future
+  //
+  // See https://issues.apache.org/jira/browse/KAFKA-13143 for details
```

Comment:
My preference would be to remove the handler completely. We can always bring it back in the future.

Reply:
Will remove the code.